### PR TITLE
Issue #929: Add a quick search field in the "Add block" dialog

### PR DESCRIPTION
**Pull request by docwilmot**
- Also added zebra striping.
- Also changed to sort by block title.

Fix for https://github.com/backdrop/backdrop-issues/issues/929

### CODE REVIEW

**Comment by docwilmot** [comment]
default layout localhost
default layout localhost 1

File: core/modules/layout/css/layout.admin.css [hunk 1]
```diff
@@ -154,4 +154,13 @@
 /* Layout Misc */
 #layout-edit h1 {
   margin: 1em 0 .6em;
-}
\ No newline at end of file
+}
+
+.layout-block-add-row.form-item {
+  padding:5px;
```
Comment (quicksketch) [inline core/modules/layout/css/layout.admin.css]: Nitpick: space missing after colon.

**Comment by docwilmot** [comment]
updated css to look like views dialog for consistency

qsearch

**Comment by quicksketch** [comment]
Nice, looks great! I ran into a few issues and filed a PR against your branch at https://github.com/docwilmot/backdrop/pull/1.

**Comment by quicksketch** [comment]
@docwilmot Do you think any other changes are needed here? I'd say give it one more review to make sure it works and then this is ready to merge.

**Comment by docwilmot** [comment]
Gave it another good look; can't think of anything.
Do I rebase and push again? That would lose your contribution, no? Otherwise, merge away!

**Comment by quicksketch** [comment]
Great, thanks! Merged. Awesome! :smile:

**Pull request merged**